viewer-config-persistence/src/main/resources/scripts/delete_orphan_featuretypes.sql

```diff
@@ -0,0 +1,9 @@
+
+delete from application_layer_attributes where attribute_ in (select id  from configured_attribute where feature_type in (select id from feature_type where id not in (select feature_type from feature_source_feature_types)));
```

Comment:
do these updates+ deletes work on oracle + sqlserver; we have some oracle users and at least sqlserver user (for which  I'm pretty sure this problem does not exist, but I we can try on the OIGS database in-house

Reply:
Oracle works